website_sale_product_subscribe_notify/models/product_template.py

```diff
@@ -24,17 +24,19 @@ def write(self, vals):
                     partner_ids = rec.get_product_category_followers_ids()
                     partners = rec.env['res.partner'].browse(partner_ids)
                     list_price = "%d" % int(rec.list_price)
+                    multi_partner_id = ''
                     for partner in partners:
-                        ctx.update({
+                        multi_partner_id = multi_partner_id + str(partner.id) + ','
```

Comment:
Maybe `','.join([partner.id for partner in multi_partner_id]))` a better option? Otherwise the `multi_partner_id` always end with a ','

Reply:
Also in this case, we can assign the above line directly in the `ctx.update`, we want to rename the `partner_id` to `partner_ids` so that we know there are multiple ids passing to this field 
i.e. `'partner_ids': ','.join([partner.id for partner in multi_partner_id])`